ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/PendingTransaction.java

```diff
@@ -31,8 +31,8 @@
 public abstract class PendingTransaction
     implements org.hyperledger.besu.datatypes.PendingTransaction {
   static final int NOT_INITIALIZED = -1;
-  static final int FRONTIER_AND_ACCESS_LIST_SHALLOW_MEMORY_SIZE = 888;
-  static final int EIP1559_AND_EIP4844_SHALLOW_MEMORY_SIZE = 1000;
+  static final int FRONTIER_AND_ACCESS_LIST_SHALLOW_MEMORY_SIZE = 904;
```

Comment:
surprised to see this in this pr

Reply:
This is because the new guava version is adding a new field to the class returned by `Suppliers::memoize`